src/content/blog/dotfiles.md

```diff
@@ -3,6 +3,7 @@ title: 'Managing dotfiles'
 description: 'A simple way to manage your dotfiles'
 pubDate: '2021-12-23T20:48:59+05:30'
 heroImage: '/dotfiles.jpg'
+authour: 'gtlsgamr'
```

Comment:
author not "authour"

Reply:
my bad. i wonder why my spellcheck extension didn't detect it.